massa-bootstrap/src/client.rs

```diff
@@ -329,7 +330,8 @@ async fn connect_to_server(
     let mut connector = establisher
         .get_connector(bootstrap_settings.connect_timeout)
         .await?; // cancellable
-    let socket = connector.connect(*addr).await?; // cancellable
+    let socket = <Limiter>::new(bootstrap_settings.max_bit_read_write.into())
```

Comment:
according to the doc of the limiter crate (https://github.com/adrien-zinger/async-speed-limit#usage), the bandwidth limit is given in bytes per second (not bits per 100ms)

Reply:
Note that "KiB" means kibibyte (1024 bytes)

Reply:
I see that the default refill period is 0.1s, can you link where the doc say 1s please? I should have missed that part

Reply:
I found the comment. Isn't it a strange difference with the documentation that follow?

Reply:
I'm not sure. If the documentation is inconsistent, we should do a measurement on a simple localhost socket connected to a random listener

Reply:
I confirm this value is in Bytes per second: 

https://github.com/tikv/async-speed-limit/blob/master/src/limiter.rs#L29

https://github.com/tikv/async-speed-limit/blob/master/src/limiter.rs#L39-L42